### PR TITLE
Fix show all and DAO votes

### PR DESCRIPTION
**Pull request by CAPtheorem**
1/ Flips Show all logic per re-opened #378 
2/ Enables "You have voted thank you logic" on the DAO per #429 by checking vote status
3/ Adds % to APR
4/ Fixes the NFT screen title error (DAO->NFT)
5/ Simplifies the layer and chain labeling
6/ Reconciles chain switching messages

### CODE REVIEW

File: packages/omgx/wallet-frontend/src/components/Proposal/Proposal.js [hunk 1]
```diff
@@ -83,6 +83,9 @@ function Proposal({
                 if(proposal.state !== 'Active') {
                     return;
                 }
+                if(proposal.hasVoted) {
```
Comment (sk-enya) [inline packages/omgx/wallet-frontend/src/components/Proposal/Proposal.js:86]: With early return code at this lie will not get execute.

I mean the user won't be able to see the details text: I guess we need to remove check at the bottom or this early return.

https://github.com/omgnetwork/optimism/pull/434/files#diff-32b2019ca228a53023b0b623415eda8d05399a80c13edfa4a1338fd66c4e0cd3R134

File: packages/omgx/wallet-frontend/src/components/listFarm/listFarm.js [hunk 2]
```diff
@@ -218,7 +218,7 @@ class ListFarm extends React.Component {
 
             <S.GridItemTag item xs={4} md={1.7}>
               {isMobile ? (
-                <Typography variant="overline" sx={{opacity: 0.7}}>APR</Typography>
+                <Typography variant="overline" sx={{opacity: 0.7}}>APR &#x25;</Typography>
```
Comment (sk-enya) [inline packages/omgx/wallet-frontend/src/components/listFarm/listFarm.js:221]: Is there any reason that we have to use hex of % symbol?

**Review by sk-enya — APPROVED** [review]

File: packages/omgx/wallet-frontend/src/components/listFarm/listFarm.js (hunk 2 above)
Comment (CAPtheorem) [reply packages/omgx/wallet-frontend/src/components/listFarm/listFarm.js:221]: probably not - please change to %

File: packages/omgx/wallet-frontend/src/components/Proposal/Proposal.js (hunk 1 above)
Comment (CAPtheorem) [reply packages/omgx/wallet-frontend/src/components/Proposal/Proposal.js:86]: we only need to show the vote buttons IF (the proposal is active) && (the person has not yet voted) so it looks ok

File: packages/omgx/wallet-frontend/src/components/listFarm/listFarm.js (hunk 2 above)
Comment (sk-enya) [reply packages/omgx/wallet-frontend/src/components/listFarm/listFarm.js:221]: okay

**Review by sk-enya — APPROVED** [review]

**Pull request merged**